site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -184,6 +184,10 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.mutatingWebhook.disableCaBundle`      | **Deprecated**. Use `agones.extensions.mutatingWebhook.disableCaBundle` instead. Disable ca-bundle so it can be injected by cert-manager                                                                                                                                                                                                                                                          | `false`                                   |
 | `agones.controller.allocationBatchWaitTime`              | Wait time between each allocation batch when performing allocations in controller mode                                                                                                                                                                                                                           | `500ms`                                   |
 | `agones.controller.topologySpreadConstraints`                           | Ensures better resource utilization and high availability by evenly distributing Pods in the agones-system namespace                                                                                                                                                     | `{}`                                   |
+| `agones.controller.maxCreationParalellism`                           | Maximum number of parallelizing creation calls in GSS controller                                                                                                                                                     | `16`                                   |
+| `agones.controller.maxGameServerCreationsPerBatch`                           | Maximum number of GameServer creation calls per batch                                                                                                                                                      | `64`                                   |
+| `agones.controller.maxDeletionParallelism`                           | Maximum number of parallelizing deletion calls in GSS                                                                                                                                                      | `64`                                   |
+| `agones.controller.maxGameServerDeletionsPerBatch`                           | Maximum number of GameServer deletion calls per batch                                                                                                                                                      | `64`                                   |
```

Comment:
These should only be visible after next release, so the whole configurations need to be wrapped by 
```
{{% feature publishVersion="1.43.0" %}}
...
{{% /feature %}}
```

For example, https://github.com/googleforgames/agones/blame/26b17c389a466097353e56cd6d82f73f2fd32516/site/content/en/docs/Installation/Install%20Agones/helm.md#L151-L226